README.md

```diff
@@ -157,105 +157,112 @@ it will also import the Player constructor directly:
-    + [getPlayed](#getplayed-promisearray-error)
-    + [getSeekable](#getseekable-promisearray-error)
-    + [getSeeking](#getseeking-promiseboolean-error)
-    + [getTextTracks](#gettexttracks-promiseobject-error)
-    + [getVideoEmbedCode](#getvideoembedcode-promisestring-error)
-    + [getVideoId](#getvideoid-promisenumber-error)
-    + [getVideoTitle](#getvideotitle-promisestring-error)
-    + [getVideoWidth](#getvideowidth-promisenumber-error)
-    + [getVideoHeight](#getvideoheight-promisenumber-error)
-    + [getVideoUrl](#getvideourl-promisestring-privacyerrorerror)
-    + [getVolume](#getvolume-promisenumber-error)
-    + [setVolume](#setvolumevolume-number-promisenumber-rangeerrorerror)
-    + [setTimingSrc](#settimingsrctimingobject-timingobject-options-timingsrcconnectoroptions-promisetimingsrcconnector)
-    + [getQualities](#getqualities-promiseobject-error)
-    + [getQuality](#getquality-promisestring-error)
-    + [setQuality](#setqualityquality-string-promisestring-typeerrorerror)
-    + [getCameraProps](#getcameraprops-promiseobject-error)
-    + [setCameraProps](#setcamerapropscameraprops-object-promiseobject-rangeerrorerror)
-* [Events](#events)
-    + [play](#play)
-    + [playing](#playing)
-    + [pause](#pause)
-    + [ended](#ended)
-    + [timeupdate](#timeupdate)
-    + [progress](#progress)
-    + [seeking](#seeking)
-    + [seeked](#seeked)
-    + [texttrackchange](#texttrackchange)
-    + [chapterchange](#chapterchange)
-    + [cuechange](#cuechange)
-    + [cuepoint](#cuepoint)
-    + [volumechange](#volumechange)
-    + [playbackratechange](#playbackratechange)
-    + [bufferstart](#bufferstart)
-    + [bufferend](#bufferend)
-    + [error](#error)
-    + [loaded](#loaded)
-    + [durationchange](#durationchange)
-    + [fullscreenchange](#fullscreenchange)
-    + [qualitychange](#qualitychange)
-    + [camerachange](#camerachange)
-    + [resize](#resize)
-    + [enterpictureinpicture](#enterpictureinpicture)
-    + [leavepictureinpicture](#leavepictureinpicture)
-    + [remoteplaybackavailabilitychange](#remoteplaybackavailabilitychange)
-    + [remoteplaybackconnecting](#remoteplaybackconnecting)
-    + [remoteplaybackconnect](#remoteplaybackconnect)
-    + [remoteplaybackdisconnect](#remoteplaybackdisconnect)
-    + [interactivehotspotclicked](#interactivehotspotclicked)
-    + [interactiveoverlaypanelclicked](#interactiveoverlaypanelclicked)
-* [Dev Options](#dev-options)
-    + [prefer_mms](#prefer_mms)
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Create a Player](#create-a-player)
+  - [Existing embed](#existing-embed)
+  - [Create an embed](#create-an-embed)
+- [Embed Options](#embed-options)
+- [Methods](#methods)
+  - [static isVimeoUrl(url: string): boolean](#static-isvimeourlurl-string-boolean)
```

Comment:
What do you think about moving this to the bottom of the list, or lower down? It might be a good idea to keep the more commonly used methods at the top.